Week1/subway/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+  <head></head>
+  <style>
+	#map {
+	   height: 100%;
+	 }
+  </style>
+  <body>
+	<div id="map"></div>
+  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script>
+	let map;
+	function getLatLng(data){
+		//entrances
+		const entrances = data[0];
+		let entlocations = [];
```

Comment:
This can actually be a `const` since it's a pointer. You aren't ever setting `entlocations` into a new array.